OpenSim/Simulation/SimbodyEngine/Joint.cpp

```diff
@@ -551,18 +576,22 @@ void Joint::updateFromXMLNode(SimTK::Xml::Element& aNode, int versionNumber)
     //bool converting = false;
     if (documentVersion < XMLDocument::getLatestVersion()){
         if (documentVersion<30500){
-            XMLDocument::renameChildNode(aNode, "location", "location_in_child"); 
-            XMLDocument::renameChildNode(aNode, "orientation", "orientation_in_child");
+            XMLDocument::renameChildNode(aNode, "location",
+                                         "location_in_child"); 
+            XMLDocument::renameChildNode(aNode, "orientation",
+                                         "orientation_in_child");
         }
-        // Version 30501 converted Connector_Body_ to Connector_PhysicalFrame_
+        // Version 30501 converted Socket_Body_ to Socket_PhysicalFrame_
         if (documentVersion < 30501) {
-            // Handle any models that have the Joint connecting to Bodies instead
+            // Handle any models that have the Joint connecting to Bodies
+            // instead
             // of PhyscialFrames
-            XMLDocument::renameChildNode(aNode, "Connector_Body_",
-                                                "Connector_PhysicalFrame_");
+            XMLDocument::renameChildNode(aNode, "Socket_Body_",
```

Comment:
I'm not totally clear on what we're claiming to support or what the intention of this change is, but it seems like we need to increment the version number, leave this code as it is, and write a new `if` block containing `XMLDocument::renameChildNode(aNode, "Connector_PhysicalFrame_", "Socket_PhysicalFrame_");`, etc. Because we had the Developer's Workshop, I think we need to assume there are model files out there containing "Connector" nodes.